README.md

```diff
@@ -1 +1,43 @@
-# To-Do-list-list-structure
\ No newline at end of file
+# To Do List
+
+In this project, I built a simple HTML list of To Do tasks. This simple web page will be built using webpack and served by a webpack dev server.
+
+
+## Built With
+
```

Comment:
- [ ] Please add the `Getting Started` section with the instructions on how to clone your repo, install dependencies libraries `npm install`, run your build `npm run build`, and run your dev server` npm run start`. To make your readme file professional and user-friendly.